### PR TITLE
Release SDK v0.1.365

### PR DESCRIPTION
**Pull request by Ankit152**
Signed-off-by: Ankit Kurmi <akurmi@redhat.com>

Jira: [OCM-1791](https://issues.redhat.com/browse/OCM-1791)

Follow up: https://github.com/openshift-online/ocm-sdk-go/pull/824

### CODE REVIEW

**Review by ciaranRoche — CHANGES_REQUESTED** [review]
Please have a look at https://github.com/openshift-online/ocm-sdk-go/blob/main/CONTRIBUTING.md#releasing-a-new-ocm-sdk-version you need to update the version const in version.go

File: CHANGES.md [hunk 1]
```diff
@@ -3,6 +3,10 @@
 This document describes the relevant changes between releases of the OCM API
 SDK.
 
+## 0.1.365
+- Update model version v0.0.312
+  - Added support for new endpoints and api updates for `addons_mgmt`
```
Comment (ciaranRoche) [inline CHANGES.md:8]: can you be specific as to what endpoints were added please :pray:

Comment (Ankit152) [reply CHANGES.md]: specified all the changes that were made 🙂

**Review by ciaranRoche — APPROVED** [review]
/lgtm

**Pull request merged**